assets/sass/05-blocks/group/_editor.scss

```diff
@@ -36,8 +36,12 @@
 		}
 	}
 
-	.wp-block-group__inner-container > *:last-child {
-		margin-bottom: 0;
+	.wp-block-group__inner-container {
+		@include innerblock-margin-clear();
+
+		> *:last-child {
+			margin-bottom: 0;
+		}
```

Comment:
I think this can be removed, it was added to fix the spacing issues but the rule from the mixin should address it.

Reply:
Cool, thanks. Removed in 20c899d.